Doc/library/cmdlinelibs.rst

```diff
@@ -1,7 +1,7 @@
 .. _cmdlinelibs:
 
 ********************************
-Command Line Interface Libraries
+Command-line interface libraries
```

Comment:
Command-Line Interfaces are commonly initialised as CLI(s), so I think caps here are reasonable:
```suggestion
Command-Line Interface libraries
```

Reply:
And https://en.wikipedia.org/wiki/Command-line_interface